src/components/App/_globalStyles.scss

```diff
@@ -4,6 +4,14 @@ body {
   font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, "Helvetica Neue", Arial, sans-serif;
   line-height: 1.6;
   margin: 0;
-  min-width: $app_min_width;
+  min-width: $app-min-width;
```

Comment:
I put a note in the readme about naming conventions for CSS and updated underscores to dashes in variables for consistency.